x-pack/plugins/security/server/routes/analytics/authentication_type.test.ts

```diff
@@ -312,21 +325,95 @@ describe('POST /internal/security/analytics/_record_auth_type', () => {
       });
 
       const response = await routeHandler(getMockContext(), request, kibanaResponseFactory);
+
       expect(response.status).toBe(200);
+
       expect(response.payload).toEqual({
         timestamp: initialTimestamp,
         signature: '46d5841ad21d29ca6c7c1c639adc6294c176c394adb0b40dfc05797cfe29218e',
       });
+
       expect(response.payload.signature).not.toEqual(initialSignature);
 
       expect(routeParamsMock.analyticsService.reportAuthenticationTypeEvent).toHaveBeenCalledTimes(
         1
       );
+
       expect(routeParamsMock.analyticsService.reportAuthenticationTypeEvent).toHaveBeenCalledWith({
         authenticationProviderType: 'http',
         authenticationRealmType: 'native',
         httpAuthenticationScheme: 'Custom',
       });
     });
   });
+
+  describe('logApiKeyWithInteractiveUserDeprecated', () => {
+    it('should log a deprecation warning if API key is being used for access via a web browser', async () => {
+      const request = httpServerMock.createKibanaRequest({
+        headers: { authorization: 'ApiKey' },
```

Comment:
__nit:__ just to make sure we properly parse full header value (same for the test with `Basic`).
```suggestion
        headers: { authorization: 'ApiKey xxxx' },
```